Auth.php

```diff
@@ -32,6 +37,7 @@ public function __construct(Model $userModel = null)
         }
 
         $this->userModel = $userModel;
+        $this->fallbackAuth = new \Piwik\Plugins\Login\Auth();
```

Comment:
Is there like a difference between extending and forwarding all calls? All good, just wondering about it :)